### PR TITLE
Multipackage updates to docs

### PR DESCRIPTION
**Pull request by harveydoodle**

### CODE REVIEW

**Review by jhrams — APPROVED** [review]

File: swagger.yml [hunk 1]
```diff
@@ -580,7 +584,7 @@ definitions:
                   example: 2.5
                 unit:
                   type: string
-                  description: The unit of measurement for the package. Can be either `kg` or `lb`.
+                  description: The unit of measurement for the package. Can be either `kg`, `g` or `lb`.
```
Comment (helenjw) [inline swagger.yml]: we still only accept kg or lb

File: swagger.yml [hunk 2]
```diff
@@ -265,13 +265,13 @@ paths:
         200:
           schema:
             $ref: "#/definitions/Order"
-          description: 'Successfull response'
+          description: 'Successful response'
```
Comment (helenjw) [inline swagger.yml:268]: A successful response on order create now looks like this:
```
  {
	"id": "ESN9NY5IE6OU",
	"packages": [
		{
			"trackingNumber": "FTWSSSZ7UJEB",
			"sizeOptions": {
				"length": 0,
				"width": 0,
				"height": 0,
				"unit": "inch"
			},
			"weightOptions": {
				"weight": 1,
				"unit": "lb"
			}
		}
	]
}
```

File: swagger.yml [hunk 3]
```diff
@@ -286,7 +286,7 @@ paths:
...
   /orders/{id}/packages:
     x-summary: Update the packages in an order along with their dimensions and weight.
```
Comment (helenjw) [inline swagger.yml]: I think we should also add somewhere in the docs that the user cant change the number of packages

File: swagger.yml [hunk 4]
```diff
@@ -881,6 +881,62 @@ definitions:
...
+                - unit
+              properties:
+                weight:
+                  type: number
+                  format: double
+                  description: The weight of the package.
+                  example: 2.5
+                unit:
+                  type: string
+                  description: The unit of measurement for the package. Can be either `kg`, `g` or `lb`.
```
Comment (helenjw) [inline swagger.yml]: remove 'g'

File: swagger.yml [hunk 5]
```diff
@@ -636,7 +640,7 @@ definitions:
               example: 2.5
             unit:
               type: string
-              description: The unit of measurement for the package. Can be either `kg` or `lb`.
+              description: The unit of measurement for the package. Can be either `kg`, `g` or `lb`.
```
Comment (helenjw) [inline swagger.yml]: remove g

**Comment by helenjw** [comment]
Oh, we also need to update the return payload of the tracking backend. It will now include an extra field "trackingNumber", and has a query param showPackageTracking
```
{
			"eventName": "GEOCODED",
			"timestamp": 1679037498774,
			"trackingNumber": "OKWIVCXADDHW",
			"eventDescription": "Address is valid"
}
```

**Review by xenyal — APPROVED** [review]

File: swagger.yml [hunk 6]
```diff
@@ -355,6 +355,11 @@ paths:
           description: The id of the order for which tracking information is being requested.
           required: true
           type: string
+        - name: showPackageTracking
+          in: query
+          description: Flag for returning the package-level tracking info in response
+          required: true
```
Comment (helenjw) [inline swagger.yml]: optional, default false

**Review by helenjw — CHANGES_REQUESTED** [review]

File: swagger.yml (hunk 6 above)
Comment (helenjw) [reply swagger.yml]: Also, Tracking Response.timeline now includes a new field `trackingNumber` which needs to be represented in the docs

File: swagger.yml [hunk 7]
```diff
@@ -877,6 +944,62 @@ definitions:
         type: number
         description: The number of packages in the order.
         example: 3
+      packages:
```
Comment (helenjw) [inline swagger.yml]: I don't think the tracking response gives back the package details. It just gives the package count

File: swagger.yml [hunk 8]
```diff
@@ -910,6 +1033,10 @@ definitions:
               type: string
               description: Name of the event
               example: "SHIPMENT_PROCESSING"
+            trackingNumber:
+              type: string
+              description: Tracking number of package for the event
```
Comment (helenjw) [inline swagger.yml]: this can be orderId or package tracking number

**Review by helenjw — APPROVED** [review]

**Pull request merged**